postprocessing/visualization/receiver/src/Filters.py

```diff
@@ -132,7 +132,7 @@ def deconv(self, waveform, dt):
   
   def apply(self, wf):
     dt = wf.time[1] - wf.time[0]
-    keys = 'uvw'
+    keys = ['v{i+1}' for i in range(3)]
```

Comment:
Probably an `f` is missing. Also, this will generate `['v0', 'v1', 'v2']`, whereas below you ask for `v1, v2, v3`.